app/experimenter/targeting/constants.py

```diff
@@ -464,8 +464,11 @@ def __post_init__(self):
 NEW_USERS_WITH_INFREQUENT_USE = NimbusTargetingConfig(
     name="New users with infrequent use",
     slug="new_users_with_infrequent_use",
-    description="0 - 6 days activity in past 28 days and profile age <= 28 days",
-    targeting=(f"{PROFILE28DAYS} " "&& userMonthlyActivity|length <= 6"),
+    description="0 - 6 days activity in past 28 days and profile age < 28 days",
+    targeting=(
+        "(currentDate|date - profileAgeCreated|date) / 86400000 < 28 "
```

Comment:
I would suggest moving this calculation to a variable like `PROFILELESSTHAN28DAYS`